src/js/no-react-entry.js

```diff
@@ -17,3 +17,7 @@ require('./legacy/mega-menu.js');
 require('./legacy/sidebar-navigation.js');
 
 require('./login/login-entry.jsx');
+
+// if (location.href.endsWith('education/apply-wizard/')) {
```

Comment:
I think you can leave this uncommented and do a build type check:

`if (__BUILDTYPE__ !== 'production' && location.href.endsWith('education/apply-wizard/')) {`